Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml

```diff
@@ -83,5 +83,5 @@
   description: A plushie of the lovely Morty. It's a resilient, yet sensitive type of plush.
   components:
   - type: Sprite
-    sprite: DeltaV/Objects/Fun/toys.rsi
+    sprite: DeltaV/Objects/Fun/mortyplush.rsi
```

Comment:
You're missing the Toys folder. the path should be `DeltaV/Objects/Fun/Toys/mortyplush.rsi`

Reply:
thanks!!!